nano/core_test/active_transactions.cpp

```diff
@@ -608,17 +617,19 @@ TEST (active_transactions, vote_replays)
 	node.process_active (send2);
 	ASSERT_TRUE (nano::test::start_elections (system, node, { send2 }));
 	ASSERT_EQ (1, node.active.size ());
-	auto vote1_send2 (std::make_shared<nano::vote> (nano::dev::genesis_key.pub, nano::dev::genesis_key.prv, nano::vote::timestamp_max, nano::vote::duration_max, std::vector<nano::block_hash>{ send2->hash () }));
-	auto vote2_send2 (std::make_shared<nano::vote> (key.pub, key.prv, 0, 0, std::vector<nano::block_hash>{ send2->hash () }));
-	ASSERT_EQ (nano::vote_code::vote, node.active.vote (vote2_send2));
-	ASSERT_EQ (1, node.active.size ());
-	ASSERT_EQ (nano::vote_code::replay, node.active.vote (vote2_send2));
+
+	// vote2_send2 is a non final vote with little weight, vote1_send2 is the vote that confirms the election
+	auto vote1_send2 = nano::vote::make_final_vote (nano::dev::genesis_key.pub, nano::dev::genesis_key.prv, { send2->hash () });
+	auto vote2_send2 = std::make_shared<nano::vote> (key.pub, key.prv, 0, 0, std::vector<nano::block_hash>{ send2->hash () });
```

Comment:
Maybe for consistency we can also replace `auto vote2_send2 = std::make_shared<nano::vote>` with `nano::test::make_vote (...)`? Or ask community to do it later.

Reply:
I started doing it and then I realised this is a perfect job for someone from community who wants to contribute.
So I created 2 issues about it:
https://github.com/nanocurrency/nano-node/issues/4403
https://github.com/nanocurrency/nano-node/issues/4402

Reply:
But, you are right, I'll change this instance just for consistency in this unit test